resources/api-docs/1.0/index.md

```diff
@@ -56,6 +56,7 @@
     - [Pagination](/api-docs/{{version}}/pagination)
     - [Bookmarks](/api-docs/{{version}}/bookmark)
     - [Dashboard Widgets](/api-docs/{{version}}/dashboard-widgets)
+    - [Default Images](/api-docs/{{version}}/default-images)
```

Comment:
If the APi route is called Default Thumbnail, the doc should also be called the same thing